dialogue/TownNPCs/Mocha/RandonBananaAzuraPistachio/BananaCardinalPistachio_Mocha_Random2.json

```diff
@@ -28,12 +28,22 @@
       "Así que pásate por aquí cuando quieras, @CharacterName.",
       "Podría hasta darte algún descuento si decides alquilar un cuarto..."
     ],
+    "pl": [
+      "Zanim odziedziczyłam tą karczmę od mojego ojca, <c=hickory>Hickory</c>...",
+      "Miałam problemy rozmawiając z klientami, kiedy mu pomagałam w karczmie.",
```

Comment:
And you can remove "w karczmie" here if you want, because it doesn't sound too good when the same word is repeated too often, but it's gramatically correct as it is.